src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -11,23 +11,25 @@ redirects:
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/set-opentelemetry-new-relic
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start-guide
   - /docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-architecture-recipes/
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start
 ---
 
-OpenTelemetry is a flexible toolkit that you can implement in a variety of ways. We recommend a basic four-step approach for setting up OpenTelemetry with New Relic. Here's an overview of the process, followed by details for each step.
+OpenTelemetry is a flexible toolkit that you can implement in a variety of ways. We recommend a basic five-step approach for setting up OpenTelemetry with New Relic. Here's an overview of the process, followed by details for each step.
 
 1. [Prerequisites](#prereqs)
 2. [Instrument your service with OpenTelemetry](#instrument)
-3. [Export your telemetry data to New Relic](#export)
-4. [View your data in the New Relic UI](#view-data)
+3. [Review configurations for exporting telemetry data to New Relic](#review-settings)
+4. [Complete the export configuration steps](#export)
+5. [View your data in the New Relic UI](#view-data)
 
-## Step 1. Prerequisites [#prereqs]
+## Step 1: Prerequisites [#prereqs]
 
 First things first:
 
 * If we don't already know you, sign up for a free [New Relic account](https://newrelic.com/signup).
 * Copy your [account license key](https://one.newrelic.com/launcher/api-keys-ui.launcher).
 
-## Step 2. Instrument your service with OpenTelemetry [#instrument]
+## Step 2: Instrument your service with OpenTelemetry [#instrument]
 
 To get started, you instrument your service with OpenTelemetry. OpenTelemetry has language-specific products and SDKs to help you. Many languages offer out-the-box instrumentation for common libraries and frameworks. Each language also provides an API for further instrumenting your service manually.
```

Comment:
i can't make a proposed edit to this for some reason, but the "return here to complete" link has #export but should be #review-settings.

Reply:
@zuluecho9, great catch!! I corrected this to the actual name of the revised step: "Go to the repository for your language and follow the instructions to instrument your service. When you're done, return here to complete [Step 3: Review configurations for exporting telemetry data to New Relic](#review-settings)."